great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -449,6 +440,128 @@ class MetricDomainTypes.
+                    column_list = [
+                        quoted_name(domain_kwargs[column_name])
+                        for column_name in domain_kwargs["column_list"]
+                    ]
+                else:
+                    column_list = domain_kwargs["column_list"]
+
+                ignore_row_if = domain_kwargs["ignore_row_if"]
+                if ignore_row_if == "all_values_are_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.and_(
+                                *(
+                                    sa.column(column_name) != None
+                                    for column_name in column_list
+                                )
+                            )
+                        )
+                    )
+                elif ignore_row_if == "any_value_is_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.or_(
+                                *(
+                                    sa.column(column_name) != None
+                                    for column_name in column_list
+                                )
+                            )
+                        )
+                    )
+                else:
+                    if ignore_row_if != "never":
+                        raise ValueError(
+                            f'Unrecognized value of ignore_row_if ("{ignore_row_if}").'
+                        )
+
+                return selectable
+
+        return selectable
+
+    def get_compute_domain(
+        self,
+        domain_kwargs: Dict,
+        domain_type: Union[str, MetricDomainTypes],
+        accessor_keys: Optional[Iterable[str]] = None,
+    ) -> Tuple[Select, dict, dict]:
+        """Uses a given batch dictionary and domain kwargs to obtain a SqlAlchemy column object.
+
+        Args:
+            domain_kwargs (dict) - A dictionary consisting of the domain kwargs specifying which data to obtain
+            domain_type (str or MetricDomainTypes) - an Enum value indicating which metric domain the user would
+            like to be using, or a corresponding string value representing it. String types include "identity",
+            "column", "column_pair", "table" and "other". Enum types include capitalized versions of these from the
+            class MetricDomainTypes.
+            accessor_keys (str iterable) - keys that are part of the compute domain but should be ignored when
+            describing the domain and simply transferred with their associated values into accessor_domain_kwargs.
+
+        Returns:
+            SqlAlchemy column
+        """
+        # Extracting value from enum if it is given for future computation
```

Comment:
comment got detached

Reply:
@jcampbell Nice catch -- I placed the comment above the appropriate line.  Thanks.